client/my-sites/theme/theme-download-card/index.jsx

```diff
@@ -31,10 +30,10 @@ const ThemeDownloadCard = React.createClass( {
 			<Card className="theme-download-card">
 				<Gridicon icon="cloud-download" size={ 48 } />
 				<p>{ downloadText }</p>
-				<Button href={ downloadURI }>{ i18n.translate( 'Download' ) }</Button>
+				<Button href={ href }>{ i18n.translate( 'Download' ) }</Button>
```

Comment:
You could consider using the `localize()` HoC to go along with the ES6ification, but totally up to you.